docs/guides/bugzilla.rst

```diff
@@ -29,8 +32,11 @@ Metadata Markers
 
 
   There are several key word arguments
-  that can be passed to this: ``uncollectif``, ``forced_streams``, etc. Look into the
-  codebase for examples of their usage.
+  that can be passed to this ``BZ`` class: ``unblock``, ``forced_streams``, etc. ``uncollectif``
```

Comment:
Shouldn't it be "`unblock` takes a boolean function so that the BZ will not block if some condition is met"?